src/components/DocListing/NestedArticles.js

```
@@ -0,0 +1,498 @@
+                      __( article?.title?.rendered, 'wedocs' )
+                    ) }` }
+                  >
+                    <div
+                      className="tooltip mr-6 cursor-pointer before:max-w-xl flex items-center flex-shrink-0 text-base font-medium text-gray-700 !shadow-none z-[9999]"
+                      data-tip={ he.decode(
+                        __( article?.title?.rendered, 'wedocs' )
+                      ) }
+                    >
+                      <span
+                        className="hover:underline group-hover:text-black"
+                        dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)